submissions/SerhiiYakovenko/tiny-js-oop/index.js

```diff
@@ -0,0 +1,92 @@
+class Inhabitant {
+   constructor(species, name, gender, legs, saying) {
+     this.species = species;
+     this.name = name;
+     this.gender = gender;
+     this.legs = legs;
+     this.saying = saying;
+     this.friends = [];
+   }
+ 
+   addFriends(friends) {
+     this.friends = [...friends];
+   }
```

Comment:
As a developer I would expect the the following
```
kot.addFriends([boi, pes]);
kot.addFriends([woman]);
```
would result `kot` to have 3 freinds as method name suggests.
This won't happen.